instagram/src/components/Login/Login.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react';
+import './Login.css';
+
+class Login extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            usernameText: '',
+            passwordText: '',
+            username: '',
+        }
+    }
+
+    submitLogin = () => {
+        window.localStorage.setItem('username', this.state.usernameText);
+        this.setState({
+            ...this.state,
+            username: window.localStorage.getItem('username')
+        })
+    }
+
+    changeHandler = (event) => {
+        this.setState({
+            ...this.state,
+            [event.target.name]: event.target.value
+        }, () => {
+            if (this.state.usernameText !== '') {
+                document.querySelector('.loginbutton').style.backgroundColor = '#5097E9';
```

Comment:
_oh no, not a querySelector!_ 
It's better, at least slightly, to use [refs](https://reactjs.org/docs/refs-and-the-dom.html) if you need to do something like this to ensure that you're getting the right thing. It'd look like [this](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs). Try it out!